app/views/layouts/application.html.erb

```diff
@@ -20,9 +20,9 @@
   </head>
 
   <body>
-    <%= render "layouts/header" %>
+    <%= render "layouts/header", cached: true %>
```

Comment:
This can potentially break when header changes, Eg before and after login

Reply:
ok. Let me add cache to some other more oftenly used page